src/screens/explore-screen/tabs/MoodsTab.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { ParamListBase } from '@react-navigation/native'
+import { NativeStackNavigationProp } from '@react-navigation/native-stack'
+import { StyleSheet, View } from 'react-native'
+
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors } from 'app/utils/theme'
+
+import { TabInfo } from './TabInfo'
+
+const messages = {
+  infoHeader: 'Playlists to Fit Your Mood',
+  infoText: 'Playlists made by Audius users, sorted by mood and feel.'
+}
+
+type Props = {
```

Comment:
for my own learning, is the pattern to have `Props` the default name for all component props defined next to their component?

Reply:
Yea I see a mix of either `Props` or `[ComponentName]Props`. Not sure if we agreed on doing one vs the other